portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +24,24 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private final ArrayList<String> messages = new ArrayList<>();
```

Comment:
nit: Use the interface type for the field: `private final List<String>`

If you want, you can also make it an ImmutableList, since it's not mutable at this time (optional though):

```java
private final ImmutableList<String> messages = ImmutableList.of("Something crazy", "Something simple", "Something amazing");
```